pkg/uefi/region.go

```diff
@@ -8,38 +8,114 @@ import (
 	"fmt"
 )
 
-// Region contains the start and end of a region in flash. This can be a BIOS, ME, PDR or GBE region.
-// This value seems to index blocks of block size 0x1000
-// TODO: figure out of block sizes are read from some location on flash or fixed.
-// Right now we assume they're fixed
-
 const (
 	// RegionBlockSize assumes the region struct values correspond to blocks of 0x1000 in size
 	RegionBlockSize = 0x1000
 )
 
-// Region holds the base and limit of every type of region. Each region such as the bios region
+// FlashRegionType represents the different types possible in a flash region
```

Comment:
Put a period at the end of this sentence.

Reply:
Done